jenkins-scripts/lib/project-default-devel-homebrew-amd64.bash

```diff
@@ -88,12 +88,9 @@ fi
 
 if [[ -n "${PIP_PACKAGES_NEEDED}" ]]; then
   brew install python3
-  PIP=pip3
-  if ! which ${PIP}; then
-    PIP=${HOMEBREW_PREFIX}/opt/python/bin/pip3
-  fi
-  # TODO use a python3 venv instead.
-  ${PIP} install --break-system-packages ${PIP_PACKAGES_NEEDED}
+  python3 -m venv ${WORKSPACE}/venv
```

Comment:
Do we need to remove the existing `venv` first?

Reply:
I was just thinking the same thing: https://github.com/gazebo-tooling/release-tools/pull/1194/commits/9c96425ccb7822f1d8ac2f58be72616df85a2fe5

testing again

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_transport-ci-gz-transport14-homebrew-amd64&build=28)](https://build.osrfoundation.org/job/gz_transport-ci-gz-transport14-homebrew-amd64/28/) https://build.osrfoundation.org/job/gz_transport-ci-gz-transport14-homebrew-amd64/28/